bundlesize.config.json

```diff
@@ -10,7 +9,7 @@
     },
     {
       "path": "./@ngxs/store/fesm2015/ngxs-store.js",
-      "maxSize": "18 kB"
+      "maxSize": "19 kB"
```

Comment:
Hmmm... I wonder what increased the bundle size?
Not ideal but we may just be forced into this to support Ivy.

Reply:
It went up to 18.07

Reply:
🤣